sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -95,6 +95,8 @@ public void close() throws IOException {
     if (options != null) {
       options.getLogger().log(SentryLevel.DEBUG, "ActivityLifecycleIntegration removed.");
     }
+
+    ActivityFramesState.getInstance().close();
```

Comment:
`l`: Maybe state seems a bit weird to me. Maybe tracker fits better here?
```suggestion
    ActivityFramesTracker.getInstance().close();
```